Dockerfile.rocm.ubi

```diff
@@ -214,6 +214,15 @@ RUN --mount=type=bind,from=build_amdsmi,src=/install,target=/install/amdsmi/ \
         /install/flashattention/*.whl\
         /install/vllm/*.whl
 
+# Make sure the torch shared libraries in the virtualenv are found by the built extensions
```

Comment:
Without these symlinks, these libraries can't be resolved in the final image:
```bash
bash-5.1$ cd ${VIRTUAL_ENV}//lib/python3.12/site-packages/vllm/*.so &&\
  ldd *.so | grep -i not\ found
        libMIOpen.so.1 => not found
        libamdhip64.so.6 => not found
        libhipblas.so.2 => not found
        libhipblaslt.so.0 => not found
        libhipfft.so.0 => not found
        libhipsolver.so.0 => not found
        libhipsparse.so.1 => not found
```

Since we build using ROCm 6.2 and we install the ROCm 6.2 version of pytorch, these should be compatible.